packages/botonic-plugin-contentful/src/contentful/delivery/cache.ts

```diff
@@ -31,5 +37,6 @@ export class CachedClientApi implements ReducedClientApi {
     this.getEntry = memoize(client.getEntry, options(2))
     this.getContentType = memoize(client.getContentType, options(1))
     this.getContentTypes = memoize(client.getContentTypes, options(0))
+    this.getLocales = memoize(client.getContentTypes, options(0))
```

Comment:
This should be `client.getLocales` instead of `client.getContentTypes`, right?

Reply:
right! remove because it's now only called from CmsInfo
I added unit tests for CmsInfo